src/js/tabs/led_strip.js

```diff
@@ -507,6 +507,25 @@ led_strip.initialize = function (callback, scrollPosition) {
             if (e.originalEvent) {
                 // user-triggered event
                 const that = $(this).find('input');
+
+                //disable blink or Larson scanner, both functions are not working properly
+                if (that.is('.function-o')) {
+                    const blink = $('.checkbox .function-o');
+                    if (blink.is(':checked')) {
+                        blink.prop('checked', false);
+                        blink.change();
+                        toggleSwitch(blink, 'b');
+                    }
+                }
+                else if (that.is('.function-b')) {
```

Comment:
```suggestion
                } else if (that.is('.function-b')) {
```

Reply:
I'm sorry, I will pay attention to the brackets.